### PR TITLE
Wrap revertConnectVoltageLevelOnLine and revertCreateLineOnLine.

### PR DESCRIPTION
**Pull request by obrix**
Wrap revertConnectVoltageLevelOnLine and revertCreateLineOnLine methods from powsybl-core.

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: cpp/src/bindings.cpp [hunk 1]
```diff
@@ -634,9 +634,11 @@ PYBIND11_MODULE(_pypowsybl, m) {
             py::arg("new_line_id"), py::arg("new_line_r"), py::arg("new_line_x"), py::arg("new_line_b1"), py::arg("new_line_b2"), py::arg("new_line_g1"), py::arg("new_line_g2"),
             py::arg("line_id"), py::arg("line1_id"), py::arg("line1_name"), py::arg("line2_id"), py::arg("line2_name"), py::arg("position_percent"),
             py::arg("create_fictitious_substation"), py::arg("fictitious_voltage_level_id"), py::arg("fictitious_voltage_level_name"), py::arg("fictitious_substation_id"), py::arg("fictitious_substation_name"));
+    m.def("revert_create_line_on_line", &pypowsybl::revertCreateLineOnLine, "reverses the action done in the create_line_on_line", py::arg("network"), py::arg("lineToBeMerged1Id"), py::arg("lineToBeMerged2Id"), py::arg("lineToBeDeletedId"), py::arg("mergedLineId"), py::arg("mergedLineName"));
```
Comment (EtienneLt) [inline cpp/src/bindings.cpp]: I think it should be snake_case format for the arguments ("line_to_be_merged1_id", ... )

File: cpp/src/bindings.cpp [hunk 2]
```diff
@@ -634,9 +634,11 @@ PYBIND11_MODULE(_pypowsybl, m) {
...
 
     m.def("connect_voltage_level_on_line", &pypowsybl::connectVoltageLevelOnLine, "connect a voltage level on a line", py::arg("network"), py::arg("bbs_or_bus_id"), py::arg("line_id"),
             py::arg("line1_id"), py::arg("line1_name"), py::arg("line2_id"), py::arg("line2_name"), py::arg("position_percent"));
+    m.def("revert_connect_voltage_level_on_line", &pypowsybl::revertConnectVoltageLevelOnLine, "reverses the action done in connect_voltage_level_on_line", py::arg("network"), py::arg("line1Id"), py::arg("line2Id"), py::arg("lineId"), py::arg("lineName"));
```
Comment (EtienneLt) [inline cpp/src/bindings.cpp]: same here

**Comment by EtienneLt** [comment]
Could you add it in network.rst ? I think it is missing

**Comment by obrix** [comment]
@EtienneLt Yes it was missing indeed, I fixed it.

**Comment by EtienneLt** [comment]
> @EtienneLt Yes it was missing indeed, I fixed it.

thanks

**Review by EtienneLt — APPROVED** [review]

**Pull request merged**